README.md

```diff
@@ -7,6 +7,7 @@ Features:
 
 * OAuth user authentication against [Meta Wiki](https://meta.wikimedia.org/).
 * Internationalization with the Intuition and jQuery.i18n libraries.
+* Interface to connect and query the replica databases
```

Comment:
Could link to https://wikitech.wikimedia.org/wiki/Wiki_replicas

Reply:
Done!